tasks/main.yml

```diff
@@ -1,5 +1,7 @@
 ---
-# TODO: Consider adding "upgrade=full" to also update everything.
+- include: updates.yml
+  when: with_updates | default('0') == "1"
```

Comment:
@geerlingguy what do you think should we give an apportunity to users to pass ansible variables to packer run? Say,
```yaml
vmware_install_open_vm_tools: no
update_apt_packages: no
```
to
```json
{
          "type": "ansible",
          "playbook_file": "ansible/playbook.yml",
          "ansible_env_vars":[
            "ANSIBLE_NOCOWS=True",
            "ANSIBLE_RETRY_FILES_ENABLED=False"
          ],
          "extra_arguments": [
            "--extra-vars", "with_updates={{ user `update_apt_packages` }}"
          ]
   }
],
  "variables": {
   "update_apt_packages":"true"
  }
```

I'm asking because packer fails when using `yes/no` for default variables
```bash
virtualbox-ovf: TASK [ansible-role-packer-build-debian : Ensure host is updated.] **************
    virtualbox-ovf: fatal: [default]: FAILED! => {"failed": true, "msg": "The conditional check 'update_apt_packages' failed. The error was: error while evaluating conditional (update_apt_packages): 'yes' is undefined\n\nThe error appears to have been in '/home/ak/Dropbox/code/packer-templates/ansible/roles/ansible-role-packer-build-debian/tasks/updates.yml': line 3, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Ensure host is updated.\n  ^ here\n"}
    virtualbox-ovf:
```

It works only with `True/False` or `0/1`.

Otherwise we can stick with default `yes/no` pattern but `extra_vars` for `ansible` packer provisioner will not work.